From rdunlap@xenotime.net  Fri May  1 13:50:27 2009
From: Randy Dunlap <rdunlap@xenotime.net>
Date: Wed, 29 Apr 2009 21:04:19 -0700
Subject: USB: composite.h: mark private struct members as private:
Cc: Greg KH <gregkh@suse.de>, David Brownell <dbrownell@users.sourceforge.net>
Message-ID: <49F92343.2080503@xenotime.net>


From: Randy Dunlap <randy.dunlap@oracle.com>

Mark internal struct members as /* private: */ so that kernel-doc
won't produce warnings about missing descriptions for them.

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Acked-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/usb/composite.h |    3 +++
 1 file changed, 3 insertions(+)

--- a/include/linux/usb/composite.h
+++ b/include/linux/usb/composite.h
@@ -124,6 +124,7 @@ struct usb_function {
 	void			(*suspend)(struct usb_function *);
 	void			(*resume)(struct usb_function *);
 
+	/* private: */
 	/* internals */
 	struct list_head		list;
 };
@@ -219,6 +220,7 @@ struct usb_configuration {
 
 	struct usb_composite_dev	*cdev;
 
+	/* private: */
 	/* internals */
 	struct list_head	list;
 	struct list_head	functions;
@@ -321,6 +323,7 @@ struct usb_composite_dev {
 
 	struct usb_configuration	*config;
 
+	/* private: */
 	/* internals */
 	struct usb_device_descriptor	desc;
 	struct list_head		configs;
